functests/2_performance_update/updating_profile.go

```diff
@@ -35,8 +36,8 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 	chkCmdLine := []string{"cat", "/proc/cmdline"}
 	chkKubeletConfig := []string{"cat", "/rootfs/etc/kubernetes/kubelet.conf"}
 	chkIrqbalance := []string{"cat", "/rootfs/etc/sysconfig/irqbalance"}
-	chkHp2M := []string{"cat", "/sys/devices/system/node/node0/hugepages/hugepages-2048kB/nr_hugepages"}
-	chkHp1G := []string{"cat", "/sys/devices/system/node/node0/hugepages/hugepages-1048576kB/nr_hugepages"}
+	chkHp2M := []string{"cat", "/sys/devices/system/node/nodeN/hugepages/hugepages-2048kB/nr_hugepages"}
```

Comment:
please use longer names for identifiers (`chkHugepages2M`).

Reply:
I have removed these variable , as it's not required if we are using  func hugepagesPathForNode(...)